### PR TITLE
spider: add automation job

### PR DESCRIPTION
**Pull request by thc202**
Copy the spider automation job from the automation add-on into the
spider add-on, doing the required changes to work with the add-on
instead of core.
Only add the spider job in the automation add-on if the core spider
extension is not yet deprecated.

Part of zaproxy/zaproxy#3113.

### CODE REVIEW

**Comment by thc202** [comment]
I can provide a diff of the changes (it's rename of resource keys, use other imports…).

File: addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Sipder Automation Framework Support
+</TITLE>
+</HEAD>
+<BODY>
+<H1>Sipder Automation Framework Support</H1>
+This add-on supports the Automation Framework.
+
+<H2>Job: spider</H2>
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html:13]: Looks like a typo in the h1

File: addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">
+<HTML>
+<HEAD>
+<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=utf-8">
+<TITLE>
+Sipder Automation Framework Support
```
Comment (kingthorin) [inline addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html:6]: Sip vs spi 😉

Comment (thc202) [reply addOns/spider/src/main/javahelp/org/zaproxy/addon/spider/resources/help/contents/automation.html]: Done.

**Review by kingthorin — APPROVED** [review]
LGTM

**Review by psiinon — APPROVED** [review]

**Pull request merged**